r-package/grf/R/forest_summary.R

```diff
@@ -83,23 +83,30 @@ test_calibration <- function(forest) {
 
 #' Estimate the best linear projection of a conditional average treatment effect
 #' using a causal forest.
-#' 
+#'
 #' Let tau(Xi) = E[Y(1) - Y(0) | X = Xi] be the CATE, and Ai be a vector of user-provided
 #' covariates. This function provides a (doubly robust) fit to the linear model
-#' 
+#'
 #' tau(Xi) ~ beta_0 + Ai * beta
 #'
 #' Procedurally, we do so be regressing doubly robust scores derived from the causal
```

Comment:
92: (nitpick) Typos in here: "we do so [be]", covariates may be "[...distince The...]", and "In the event the treatment is [continous]" (this typo) appears later too)